WIP: Attempt to merge schemaorg master 2020-07-08 into BioChemEntity branch

stain opened the pull request:
See #8 and highlighted conflicts.

RichardWallis committed:
Fixed misnaming of type in examples

RichardWallis committed:
Revert PR2250 - Enum rationalise (#2443)

* Revert "Rationalise definitions of enumerations and enumeration values"

This reverts commit 97643b95a4992a999d09d822d5b28f0e27bec56e.

* Revert "Tweaked tests and added new ones to cope withe enumeration and enumerationvalue rationalisation"

This reverts commit c3d581ebee7a8f6c47b5dfa2e013c2b4880c9dfa.

Co-authored-by: Richard Wallis <rjw@dataliberate.com>

danbri committed:
Removed reverted item from release notes.

danbri committed:
Generated files for v6 release.

danbri committed:
Updated to remove references to our old branch naming system.

danbri committed:
Fixed typo: Recommendation not Recommended.

RichardWallis committed:
Fixed incorrect source link on HealthAspectEnumeration

tjann committed:
Remove overly vague medical health properties (issue 2429) (#2448)

danbri committed:
move V6 patch and medical, usageinfo drafts into master (#2456)

* Reinstated 301 redirect to https

* Fixed typo: Recommendation not Recommended.

* Add in tweaks to OPTIONS request response headers

* Drafting implementation of MediaReview for discussion. /cc #2450

* Updated mediaAuthenticityCategory label.

* Added disclaimer. /cc #2450

* updated so that label matches id.

* usageInfo and acquireLicensePage properties #2454

Co-authored-by: Richard Wallis <rjw@dataliberate.com>

Dataliberate committed:
Prepared release for v7.0

Dataliberate committed:
Sync with sdopythonapp

danbri committed:
Fixed inconsistent use of hash in release log entries.

danbri committed:
Implementation of several improvements to real estate properties.

* Add petsAllowed to ApartmentComplex
* Add petsAllowed, amenityFeature and floorSize to FloorPlan
* Create an explicit numberOfBedRooms and allow it in Accommodation, ApartmentComplex and FloorPlan.
* In ApartmentComplex and FloorPlan the value might be a range to allow describing all apartments under that entity, so the value should allow both Number and QuantitativeValue
* Add a "book a tour" (/tourBookingPage) property - url value allowed on Accommodation and ApartmentComplex

vickitardif committed:
Issue #2460: Added properties to EducationalOccupationalProgram to support cases where it is not clear the distinction between classroom-based and work-based programs. (#2461)

* Issue #2341: Added SportsTeam to the domain for gender.

* Remove file added to wrong branch.

* Issue #2460: Added properties to EducationalOccupationalProgram.

danbri committed:
/cc #2373

danbri committed:
Updated release notes.

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

gkellogg committed:
Update Gemfile.lock, fixes nokogiri warning.

RichardWallis committed:
Merge pull request #2476 from gkellogg/gem-update

Update Gemfile.lock, fixes nokogiri warning.

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

Dataliberate committed:
Add schemaorg.owl to repo
Ensure schemorg.owl & sitemap.xml are built for all deploys

RichardWallis committed:
Merge pull request #2478 from schemaorg/issue-2472

Fix schemaorg.owl missing from deploys

Dataliberate committed:
renamed:    runpythonpostprepare.sh -> scripts/runpythonpostprepare.sh

Dataliberate committed:
Pre-build of V7.0

Dataliberate committed:
Add check for releases files on deployment even wen not rebuilding static files

Dataliberate committed:
pre-build of V7.0

Dataliberate committed:
rebuild of 7.0 statics

RichardWallis committed:
Merge pull request #2479 from schemaorg/pre7.0

Fix to missing version file

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

danbri committed:
Fixed pre-launch glitches in #2373

vickitardif committed:
Issue #2486: Added startDate/endDate to Schedule (#2487)

* Issue #2341: Added SportsTeam to the domain for gender.

* Remove file added to wrong branch.

* Issue #2486: Added startDate/endDate to Schedule

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

danbri committed:
Updated to generalize from podcasts, reference DataFeed.
For #2490

danbri committed:
First cut at SpecialAnnouncement and CovidTestingFacility markup.
For #2490

danbri committed:
Updated - added properties

RichardWallis committed:
Fix to Wikipedia URL

Removed escaped characters and italic effect

danbri committed:
First cut examples. /cc #2490

danbri committed:
Updates to covid vocab documentation. /cc #2490

danbri committed:
Updates, mostly bugfixes.
Amended /category to be expected on SpecialAnnouncement
and to expect URL as possible value.
/cc #2490

vickitardif committed:
Issue #1842: Added VirtualLocation (#2488)

* Issue #2341: Added SportsTeam to the domain for gender.

* Remove file added to wrong branch.

* Issue #1842: Added VirtualLocation.

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

danbri committed:
Event-related improvements for v7.0 coronavirus updates.
/cc #1842 #2490

danbri committed:
Updated for release.

danbri committed:
Final edits for 7.0 push.

Dataliberate committed:
Fix typo

Dataliberate committed:
Ennhanced reporting

Dataliberate committed:
Sync with updated sdopythonapp

danbri committed:
Regenerated v7 release.

vickitardif committed:
Added example for the geographic region of a SpecialAnnouncement (#2495)

* Issue #2341: Added SportsTeam to the domain for gender.

* Remove file added to wrong branch.

* Added example for how to specify geo graphic region(s) for a SpecialAnnouncement.

RichardWallis committed:
Fixed missing comma in spatialCoverage example for SpecialAnnouncement

danbri committed:
Simple definition of SchoolDistrict for #2500

RichardWallis committed:
Added datePosted to SpecialAnnouncement (#2497)

Extended datePosted range to include DateTime

Co-authored-by: Dataliberate <rjw@dataliberate.com>

danbri committed:
Minor release notes for 8.0
See #2493

danbri committed:
Updated for v8 #2501.

danbri committed:
Added @ sign to "type" in example, thanks @tmarshbing!
For #2495
/cc @vtardif

danbri committed:
7.01

danbri committed:
Noted python 2.7 dependency

danbri committed:
Updated for 7.01

danbri committed:
Committing 7.01 files from release. #2501

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

danbri committed:
7.01 release files. /cc #25010

danbri committed:
Linked to google doc with draft guidelines.

This will settle into an HTML page here ASAP, but linking now for discoverability during drafting process.

RichardWallis committed:
Fixed typo in SpecialAnnouncement example

As identified in https://github.com/schemaorg/schemaorg/issues/2490#issuecomment-603739683 to issue #2490  by @Tiggerito

vickitardif committed:
Issue #2514: Added announcementLocation and updated examples (#2515)

* Issue #2341: Added SportsTeam to the domain for gender.

* Remove file added to wrong branch.

* Issue #2514: Added announcementLocation and updated examples.

danbri committed:
Adding /announcementLocation per #2514 discussion.
Thanks @tmarshbing @vholland et al

danbri committed:
Updated to match announcementLocation #2414
For #2490

danbri committed:
Updated for 7.02 release #2517 adding #2514
this is a new property /announcementLocation and
clarification on when to use it vs /spatialCoverage.

danbri committed:
Updated to use non-April date.

danbri committed:
Adding generated files (to be regenerated again).

danbri committed:
Release 7.02 files. #2501

danbri committed:
Make 3DModel a MediaObject rather  than a CreativeWork
Fixes #2518

danbri committed:
Expect webFeed on SpecialAnnouncement
Fixes #2519

danbri committed:
Adding CDC documentation prior to posting schema.
For #2521

danbri committed:
Updated for 7.03

danbri committed:
Added release notes for #2521 7.03 minor release.

danbri committed:
Renamed main type to CDCPMDRecord
For #2521

danbri committed:
Some tidying and better linking.

danbri committed:
First cut at schema definitions for #2521 COVID CDC definitions.

danbri committed:
Polish.

danbri committed:
Removed per-property boilerplate - unreadable and confusing in website ui.

danbri committed:
Linked data-and-datasets.html and cdc-covid.html docs.

danbri committed:
Release prep.

danbri committed:
7.03
For #2522

ThibaultGerrier committed:
Fix typo in versions.json (#2524)

danbri committed:
Adding cvdFacilityId and cvdFacilityCounty to CDCPMDRecord
Urgently requested for COVID-19 responses.
I will push to live site as a v2 of 7.03.

danbri committed:
Updated for cvdFacilityId and cvdFacilityCounty

danbri committed:
Fixed markup bug.

danbri committed:
7.03 (director's cut)

danbri committed:
Adding a /Hackathon type
For #2526

vickitardif committed:
Issue #2540: Made EducationalOrganization a subclass of CivicStructure (#2541)

* Issue #2341: Added SportsTeam to the domain for gender.

* Remove file added to wrong branch.

* Issue #2540: Made EducationalOrganization a subclass of CivicStructure

danbri committed:
EducationalOrganization subclassof CivicStructure
Reasonably commonsense ommission being fast-track fixed for
covid-19. Am fixing in core rather than pending due to there not
being any new vocabulary involved (so removing the #2450 file).
Fixes #2450

lucy-kind committed:
add government benefits coverage (#2538)

* add government benefits coverage

* Update issue-2534.rdfa

* Update issue-2534-examples.txt

danbri committed:
Updated to use a new /jurisdiction property, and relate it
as a superproperty to legislationJurisdiction from the
Legislation work, /cc #1156.

Foor #2534

RichardWallis committed:
Added startTime / endTime to Schedule (#2494)

Co-authored-by: Dataliberate <rjw@dataliberate.com>

danbri committed:
Error in #2490 definition associated /category and /URL by domain not range.
Fixes #2536
Thanks @VladimirAlexiev

danbri committed:
Updating for a draft 7.04 release laterthis week.

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

danbri committed:
Typo

danbri committed:
Removed redeclaration of /url from #2534

danbri committed:
Fixed a bug, legislationJurisdiction was subpropertied to itself.
For #2534

danbri committed:
I commited an earlier draft version of the file somehow.

Fixed definitions.

Fixes #2526

danbri committed:
Updating for 'legislationJurisdiction' -> 'jurisdiction'

danbri committed:
Finalizing govt benefits vocabulary.

Some last minute simplifications after it was pointed out we
had relevant existing vocabulary.

danbri committed:
More cleaning out of older complex design.

danbri committed:
Version 7.04.

danbri committed:
Removed benefitsType (cleanup).

danbri committed:
7.04

danbri committed:
Updated example to match vocab.

danbri committed:
Fixed benefits example and list of associated terms. Fixed domain/range bug.

danbri committed:
7.04 release.

danbri committed:
7.04

danbri committed:
7.04

danbri committed:
7.04

danbri committed:
First cut at #2506 shipping for Pending

lucy-kind committed:
Add coverage for non-profit organizations (#2545)

* Create issue-2543.rdfa

adds coverage for non-profit organizations

* Create issue-2543-examples.txt

adds coverage for non-profit organizations

* Update issue-2543.rdfa

* Update issue-2543.rdfa

Add NL non-profit types to NonprofitType enum list

danbri committed:
Wrapping up for v8 release

philbarker committed:
Adding eligibilityToWorkRequirement property to JobPosting for visa and citizenship requirements (#2492)

danbri committed:
Adds rest of #2528

danbri committed:
doesNotShip had wrong range
also rewording 'vendor' as 'merchant' for readibility

danbri committed:
Another typo: s/URL/Text/

danbri committed:
Implemention of 2 EIDR properties #2469

This file got dropped accidentally during the fast-tracking of
COVID-changes last month. See github issue for history; originally
modeled with a single property but using two now to better capture
the semantics of EIDR.

danbri committed:
added EIDR

danbri committed:
v8

danbri committed:
v8

danbri committed:
Fixed markdown syntax bug.

danbri committed:
Second occurence of the same markdown bug.

danbri committed:
Fixed mis-capitalized link to nonprofitStatus; thanks @tmarshbing!

philbarker committed:
Rationalize/simplify education-related properties (#2428)

* Adding properties for teaches and assesses to CreativeWork and EducationEvent; extending range of educationLevel, adding guidance to educationalAlignment, amending examples.

* referencing issue 2427

* referencing learning outcomes in defitions to fulfil requirement of #2432

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

danbri committed:
Typo

danbri committed:
v8.0 release

lucy-kind committed:
Update issue-2543.rdfa (#2567)

add nested structure for nonprofit type enumeration by country

danbri committed:
we don't use release-named branches any more

danbri committed:
Static snapshot of homepage

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

Dataliberate committed:
Temp move to NDBSHARED pagestore and static homepage

Dataliberate committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

Dataliberate committed:
Fixed typos

Dataliberate committed:
Added CORS and json-ld link headers to statics

RichardWallis committed:
Added access-control-expose-headers: Link header

Dataliberate committed:
Enhancements for staticbuild

Dataliberate committed:
Merge branch 'master' into staticbuild

Dataliberate committed:
Merge branch 'staticbuild'

Dataliberate committed:
Remove temp file from repo

Dataliberate committed:
Remove need for -d mode

Dataliberate committed:
Removed dependancy on python 3

RichardWallis committed:
Correction to webschemas-g ID

Dataliberate committed:
Removed auto_scaling values from webschemasorg & schemaorg yamls tweaked syntax of handlers

Dataliberate committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

Dataliberate committed:
Syncronised local handlers with handlers

Dataliberate committed:
Reorderd to make it more edit friendly

Dataliberate committed:
Updates to enable building of differing release versions

Dataliberate committed:
Added README text for staticbuild

RichardWallis committed:
Merge pull request #2590 from schemaorg/staticbuild2

Version specific static builds + READMEs

Dataliberate committed:
Removed references to content negotiation on term pages

RichardWallis committed:
ttl support (#2574)

* Move to ttl input files

* Replaced dc:source with schema:source

* Removed tests for rdfa

* Changes for move from rdfa to ttl

* Support ttl source files

* regenerated output file

* Removed rdfa checks from travis

* Added issue-2543.ttl to match updated issue-2543.rdfa in master to aid merge

* Revert "Added issue-2543.ttl to match updated issue-2543.rdfa in master to aid merge"

This reverts commit f05798ecb5facc2345e5946c55ef5e3585cee55a.

* Revert "Added issue-2543.ttl to match updated issue-2543.rdfa in master to aid merge"

* Allow to bypass version check for the special case of 'tmp'

* ync with sdopythonapp

* Corrected inavertant delete

Co-authored-by: Dataliberate <rjw@dataliberate.com>

RichardWallis committed:
Wrapped true in '' in http headers

RichardWallis committed:
Resolve display of URL in dowloads section of developers page.
Issue (#2592)

RichardWallis committed:
Set turtle mime_type to text/turtle

RichardWallis committed:
Ensure handlers.yaml & handlers-local.yaml are always built from templates

RichardWallis committed:
Updates to yaml for version paths and Access-Control-Allow-Origin: "*"

RichardWallis committed:
Set correct mime_type for /docs/jsonldcontext.jsonld

RichardWallis committed:
Set mime_type for jsonldcontext variations

danbri committed:
Fixed a 404 typo

Dataliberate committed:
Added setting of FORCEINDEXPAGES to supresses <meta name="robots" content="noindex"> in static build

Dataliberate committed:
Sync with sdopythonapp

Dataliberate committed:
Sync with sdopythonapp

danbri committed:
variant and ProductGroup changes for #1797 and #2597
including minimal linking into core via isVariantOf

danbri committed:
Noted the inverseOf and new domain/range values.

danbri committed:
Added missing namespace declarations.

danbri committed:
Define both sides of the inverseOf here.
Needed to pass sanity checks that there are even number of inverseOf claims.
/cc #1797

danbri committed:
Added productGroupID to identify groups
Hyperlinked 'size' in ProductGroup definition.
Removed accidental quote chars.
/cc #1797

danbri committed:
Added inProductGroupWithID, to leave open possibility of groups in groups.
/cc #1797

danbri committed:
Fixed rdfs:label

danbri committed:
Reworded /Collection to be more domain-neutral (e.g. for subtyping).
Fixes #2563

danbri committed:
First cut at a definition adapting Collection for Products
/cc #2506

Dataliberate committed:
Added environ value to enable [more..] tab display on static build.

vickitardif committed:
Issue #2599: Added byMonthWeek and an example (#2606)

* Issue #2341: Added SportsTeam to the domain for gender.

* Remove file added to wrong branch.

* Issue #2486: Added startDate/endDate to Schedule

* Issue #2599: Added byMonthWeek to Schedule

* Removed stale RDFa files to sync with schema.org main.

* Issue #2599: Added byMonthWeek to Schedule

* Fixed formatting

Dataliberate committed:
Rationalisation of enum definitions

Dataliberate committed:
Refine enum properties display

RichardWallis committed:
Merge pull request #2614 from schemaorg/enumupdate

Rationalisation of enum definitions

alex-jansen committed:
Fix typo (#2616) (#2617)

* Fix typo (#2616)

* Fix broken link

Dataliberate committed:
Corrected duration examples see issue #2608

Gnomus042 committed:
Various examples added (#2620)

* Various examples added

* cutOffTime in OfferShippingDetails: Date substituted with Time

Co-authored-by: Anastasiia <anastasiia.byvsheva@gmail.com>

lucy-kind committed:
Create issue-2611.ttl (#2612)

danbri committed:
Adding LearningResource based on 2016 design and recent discussion.
/cc #1401

danbri committed:
Adds a Quiz type, as a kind of LearningResource

danbri committed:
9.0

danbri committed:
Fixed prefixes.

danbri committed:
Removed dc: prefix.

danbri committed:
@lucy-kind suggests a comma

RichardWallis committed:
Update physical-activity-and-exercise.ttl

Fix for issue #2268

RichardWallis committed:
Update schema.ttl

Updated RadioStation description see issue #2102

RichardWallis committed:
Fixed typo in RadioStation comment

danbri committed:
Update Gemfile.lock

Update from rack (~> 2.0) to rack (>= 2.2.3)
/cc @rjw @gkellogg

danbri committed:
Update Gemfile.lock

Bumped another rack reference to 2.2.3 
/cc @rjw @gkellogg

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

dependabot[bot] committed:
Bump rack from 2.2.2 to 2.2.3 in /scripts (#2633)

Bumps [rack](https://github.com/rack/rack) from 2.2.2 to 2.2.3.
- [Release notes](https://github.com/rack/rack/releases)
- [Changelog](https://github.com/rack/rack/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rack/rack/compare/v2.2.2...2.2.3)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

danbri committed:
Allowing includesObject on ProductCollection. #2605

alex-jansen committed:
Fix Issue#2629 (#2630)

* fix broken link (issue #2622)

* Change /cutoffTime type from Text to Time

* add property isUnlabelledFallback to class DeliveryTimeSettings

* fix issue #2629

danbri committed:
All ProductCollections are now also products. #2605

danbri committed:
Changes towards v9 /cc #2634

lucy-kind committed:
issue-2604 reformat status type enumeration (#2609)

* Create issue-2604.ttl

Nest status type enumeration lists under StatusEnumeration

* Update issue-2604.ttl

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

danbri committed:
Changes need for #2604 to make #2609 work:

Pending should not re-define core terms since definitions are additive
across data/*.ttl files. Instead we tweaked the core minimally to
mention the new supertype.

/cc @lucy-kind @rjw

danbri committed:
Updated for v9 release. #2634

danbri committed:
Updating towards v9 release.

StatusEnumeration is now in Core.

danbri committed:
Added Turtle version of #2465 PR for #1755
Adds BoatTerminal, BoatReservation, BoatTrip

danbri committed:
Restored StatusEnumeration after git reverted it.

danbri committed:
Added BoatTrip etc for #1755

danbri committed:
Multi-line quotes with whitespace need triple-quoting.

AlasdairGray commented:
I'm confused. I've checked out this branch and don't see any conflicts highlighted. However, it seems that this branch does not include any merging from the Bioschemas branches. Am I looking at the right thing?

When this branch is merged in, will it then work with the new schemaorg way of building rather than the old? Have you been able to get that working? I always get an error due to the rdflib (Issue 10 on schemaorg/sdopythonapp repo ([link](https://github.com/schemaorg/sdopythonapp/issues/10))).

AlasdairGray commented:
The changes we made are summarised here (as far as I can tell):

sdoapp.py: add `'bio'` to the list of enabled extensions
```
ENABLED_EXTENSIONS = [ATTIC, 'auto', 'bib', 'health-lifesci', 'pending', 'meta', 'iot', 'bio' ]
```

sdoconfig.json: add a descriptor for the bio extension
``` json
{
           "@type": "VocabDescriptor",
           "id": "http://bio.schema.org",
           "extension": "bio",
           "name": "Biology and the Life Sciences Section",
           "softwareVersion": "1.0",
           "linktext": "Defined in the %s section.",
           "disambiguatingDescription": "terms relating to biology and the life sciences.",
           "comment": "The Biology and Life Sciences Section within [schema.org](http://schema.org) defines terms such as [[BioChemEntity]], [[Gene]], [[MolecularEntity]], [[Protein]], and [[Phenotype]]. For more details see the Bioschemas Community [website](https://bioschemas.org/)."
},
```

sdoconfigTermsData.json: enable the various files in which we store the bioschemas proposals and the examples for these
``` json
{
            "@type": "DataDownload",
            "fileContent": "terms",
            "contentLocation": "[[SCHEMAORGLOC]]/data/ext/bio",
            "contentFile": [
                "BioChemEntity.rdfa",
                "BioChemStructure.rdfa",
                "BioSample.rdfa",
                "ChemicalSubstance.rdfa",
                "DataRecord.rdfa",
                "DNA.rdfa",
                "Enzyme.rdfa",
                "Gene.rdfa",
                "LabProtocol.rdfa",
                "MolecularEntity.rdfa",
                "Phenotype.rdfa",
                "Protein.rdfa",
                "RNA.rdfa",
                "SequenceAnnotation.rdfa",
                "SequenceMatchingModel.rdfa",
                "SequenceRange.rdfa",
                "Study.rdfa",
                "Taxon.rdfa",
                "TaxonName.rdfa"
            ]

        },
        {
            "@type": "DataDownload",
            "fileContent": "examples",
            "contentLocation": "[[SCHEMAORGLOC]]/data/ext/bio",
            "contentFile": [
                "bioChemEntity-examples.txt",
                "datarecord-examples.txt"
            ]

},
```

danbri committed:
Added missing "be". Thanks @philbarker!

danbri committed:
Omitting :audienceType (tx @philbarker)

danbri committed:
Updated likely release date.

danbri committed:
Fixed example DigitalDocumentPermission[Type]
Fixes #2631

gkellogg committed:
Bundler update (#2637)

* Update Gemfile.lock and simplify travis startup.

* Use older update of rubygems and bundler, as this environment doesn't seem to be up-to-date enough.

* Include nokogumbo gem to try to eliminate false HTML errors.

* Temporarily use rdf-rdfa from github.

* Use updated RDFa and Microdata gems and turn off SOFT_LINT to allow Ruby tests to actually fail if there's a problem!

* Turn back on SOFT_LINT, it was too good to be true :(. Errors are semantic, not syntactic.

gkellogg committed:
Gemfile update (#2643)

* Get new gem versions for RDFa and Microdata.

* Add -t option to buildreleasefiles to skip unit tests, and remove some other unused targets.

* Bash syntax error.

* Allow -t as an option to buildreleasefiles.

alex-jansen committed:
small updates to examples (issue #2639) (#2640)

* small updates to examples

* revert change of numbers to quoted numbers

* Bundler update (#2637)

* Update Gemfile.lock and simplify travis startup.

* Use older update of rubygems and bundler, as this environment doesn't seem to be up-to-date enough.

* Include nokogumbo gem to try to eliminate false HTML errors.

* Temporarily use rdf-rdfa from github.

* Use updated RDFa and Microdata gems and turn off SOFT_LINT to allow Ruby tests to actually fail if there's a problem!

* Turn back on SOFT_LINT, it was too good to be true :(. Errors are semantic, not syntactic.

Co-authored-by: Gregg Kellogg <gregg@greggkellogg.net>
Co-authored-by: Dan Brickley <danbri@google.com>

danbri committed:
Removed redundant declaration of /isAccessibleForFree on /PublicationEvent
Fixes #1344

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg into master

danbri committed:
Fixed formatting error

danbri committed:
Create stale.yml

Trying the stale issue/PR tagger /cc @RichardWallis

Dataliberate committed:
Fix error in bypass tests code.

RichardWallis committed:
Added tests for properties without domain or range values (#2652)

Co-authored-by: Dataliberate <rjw@dataliberate.com>

philbarker committed:
fixing namespace errors pointed out in issue #2485 (#2542)

danbri committed:
Slightly better "stale" message.

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg into master

danbri committed:
Fixed #2653
Thanks @krishna-pearl ! :)

RichardWallis committed:
Fix for issue (#2512) - corrected metadata definitions for LockerDelivery and ParcelService (#2655)

Co-authored-by: Dataliberate <rjw@dataliberate.com>

danbri committed:
Moving comics definitions into single bib file.

We will at some point soon migrate this into core and/or pending sections
of the site.

danbri committed:
Moved comics into bsdo-1.0.ttl

Removed @en language tags for now.
This is consistent with our other files.

There is probably a case to make for tagging everything, but
for now let's get consistency.

Dataliberate committed:
Added intercept page for retired docs/schema_org_rdfa.html
Implements issue (#2656)

danbri committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg into master

danbri committed:
Removed version info and file: URL

Dataliberate committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

Dataliberate committed:
Merge branch 'master' of https://github.com/schemaorg/schemaorg

danbri committed:
Added release files for 9.0 and updated term list.

danbri committed:
Release prep

Dataliberate committed:
Removed spurious reference to file:// subject
Fixes (#1778)

RichardWallis committed:
Rationalisation of dump files (#2654)

* Simplified and expanded dump files and updated associated documentation.
Re: issue (#2304)

* Adjust all-lays copies to account for limited file builds (as in travis)

* Updated dump file names to schemaorg-current & schemaorg-all

* Updated to create http & https versions of dumpfiles.

* Modified name of dumpfile used to reflect changes

Co-authored-by: Dataliberate <rjw@dataliberate.com>

Dataliberate committed:
Changed expected type of repetitions to QuantitativeValue
Issue (#2268)

Dataliberate committed:
Updates to releases.html for V9.0

RichardWallis committed:
Update buildreleasefiles.sh

Moved call to buildTermConfig.sh to before running unit tests
Fix for issue #2657

danbri committed:
Hyperlinked /Quiz and noted that /Collection has been clarified.

danbri committed:
Released files for v9.0 #2634

RichardWallis committed:
Fix typo in developers.tpl

Dataliberate committed:
Added dump file rationalisation to releases

danbri committed:
Updated files for v9 release

danbri committed:
Update for v9
